### DISCUSSION TITLE
✅ Ability to open several WebSSH windows

### DISCUSSION THREAD

**Discussion by egolfeld** [isontheline/pro.webssh.net#483] [Ideas]
If user wants to open several ssh connection and observe it together, he can't:

The error says: Several windows Unsupported.
<img width="1080" alt="Изображение PNG" src="https://user-images.githubusercontent.com/35452983/146833825-aad6016d-bd6a-4af6-99a1-e5ab2d99fb2f.png">

![AA677B65-736F-438C-9994-EA74E5D57F2A](https://user-images.githubusercontent.com/35452983/146833743-750e283f-fb14-454b-a4a3-226cfb13ef90.jpeg)

**Comment by isontheline** [comment]
Feature already planned here : https://github.com/isontheline/pro.webssh.net/issues/151

**Reply by BobDenny** [reply]
If you are on the same host, consider instead to use tmux. One SSH connection, many windows and panes :-)

https://github.com/tmux/tmux/wiki

**Reply by isontheline** [reply]
Sorry for the late response @egolfeld 

As @BobDenny says you can use tmux to split your logical windows or you can use Split View which is now supported since WebSSH 17 : https://github.com/isontheline/pro.webssh.net/issues/151
